include/veriblock/blockchain/temp_block_tree.hpp

```diff
@@ -53,13 +61,39 @@ struct TempBlockTree {
     return true;
   }
 
-  const block_tree_t& getStableTree() const { return tree_; }
+  void removeTempSingleBlock(const typename block_t::hash_t& hash) {
+    auto* index = getTempBlockIndex(hash);
+
+    if (index != nullptr) {
+      removeTempSingleBlock(*index);
+    }
+  }
+
+  void removeTempSingleBlock(index_t& index) {
+    // TODO: decrease counter for the previously known blocks
+    auto shortHash = tree_->makePrevHash(index.getHash());
+    auto it = temp_blocks_.at(shortHash);
```

Comment:
Assert that temp_blocks_ contains shortHash

Reply:
added